src/features/ForgotPassword/ui/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Alert, Container } from "react-bootstrap"
+import { useNavigate } from "react-router-dom"
+
+import { useForgotPasswordMutation } from "~/entities"
+import { BasicButton, BasicFormProvider, Input, isObjectEmpty, ROUTES, useCustomForm } from "~/shared"
+
+import { useForgotPasswordTranslation } from "../lib/useForgotPasswordTranslation"
+import { ForgotPasswordSchema, TForgotPasswordForm } from "../model/schemas"
+
+export const ForgotPasswordForm = () => {
```

Comment:
I think in the scope of a feature a form should be in a separate file, e.g. `ForgotPasswordForm.tsx`, not in the `index.tsx`. Because an `index` file implies the feature (user's action) itself. And this case a feature is a set of ui and models.

Reply:
Done